.npmignore

```diff
@@ -1,7 +1,5 @@
 node_modules/
 .nyc_output/
 coverage/
-src/
```

Comment:
Not really, I just thought about doing these preparations in order to get #270 merged.

I have no opinion _against_ correcting v2 prior to that (that is, to include src and test again, for a v2.1 too). So, we could make a clean 2.1.0. Shall we do it? I'd rename this PR then.